### PR TITLE
[24.1] Env vars for disable email login

### PR DESCRIPTION
**Pull request by justinegeffen**
Added disable email config to environment variables.

### CODE REVIEW

File: platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml [hunk 1]
```diff
@@ -30,6 +30,11 @@
     Your Seqera system administrator contact email.
   Value:                '`tower@your-company.com`'
 - 
+  AWS Parameter Store:    '`{prefix}/tower/disable-email`'
```
Comment (gwright99) [inline platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml:33]: You sure about this? My (functional) tower.yml is `tower.auth.disable-email`. You are missing the `auth` part in your prefix.

Generally the PS prefix path follows the structure of the tower.yml.

Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml:33]:
```suggestion
  AWS Parameter Store:    '`{prefix}/tower/auth/disable-email`'
```

**Review by llewellyn-sl — APPROVED** [review]
Made one correction. The rest LGTM, thanks!

File: platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml (hunk 1 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml]: Fixed.

**Pull request merged**